.github/workflows/create-release.yml

```diff
@@ -42,21 +42,40 @@ jobs:
     runs-on: ubuntu-latest
     needs: smoke
     steps:
+    - name: Checkout With History
+      uses: actions/checkout/@v2
+      with:
+        fetch-depth: 0 # gets full history
+    - name: Compare With Previous Release
+      id: compare_previous_release
+      run: |
+        output_string="::set-output name=builder_changes"
+        if [ -z "$(git diff $(git describe --tags --abbrev=0) -- builder.toml)" ]
+        then
+          echo "${output_string}::false"
+        else
+          echo "${output_string}::true"
+        fi
     - name: Checkout
+      if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
       uses: actions/checkout@v2
-    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* || true
+    - if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* || true
     - name: Reset Draft Release
       id: reset
+      if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
       uses: paketo-buildpacks/github-config/actions/release/reset-draft@main
       with:
         repo: ${{ github.repository }}
         token: ${{ github.token }}
```

Comment:
Will this be necessary once we are no longer creating automated draft releases?

Reply:
@arjun024 and I [talked about this a bit earlier](https://github.com/paketo-buildpacks/full-builder/pull/10#discussion_r502596382). The reset action outputs the version of the draft release it deleted, and the tag action accepts that version and increments the patch number to create the new tag. 

Leaving this here + using the paketo create-release action together means that if we manually create a draft release and specify some version, we are able to intervene in the versioning that will be automatically generated by the action.

Happy to remove for simplicity as long as we still aren't caring about semver.